### PR TITLE
Fix/not null and rendered

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

File: src/main/java/io/kestra/plugin/fs/vfs/Upload.java [hunk 1]
```diff
@@ -32,12 +34,14 @@ public Upload.Output run(RunContext runContext) throws Exception {
         try (StandardFileSystemManager fsm = new StandardFileSystemManager()) {
             fsm.init();
 
+            var renderedFrom = runContext.render(this.from);
+            var renderedTo = this.to != null ? runContext.render(this.to) : renderedFrom.substring(renderedFrom.lastIndexOf('/'));
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/fs/vfs/Upload.java:38]: check if lastIndexOf == -1

File: src/main/java/io/kestra/plugin/fs/vfs/Uploads.java [hunk 2]
```diff
@@ -43,17 +48,21 @@ public Output run(RunContext runContext) throws Exception {
...
+                String renderedTo = runContext.render(this.to);
+                return VfsService.upload(
                     runContext,
                     fsm,
                     this.fsOptions(runContext),
-                    URI.create(runContext.render(fromURI)),
-                    this.uri(runContext, this.to + fromURI.substring(fromURI.lastIndexOf('/') + (this.to.endsWith("/") ? 1 : 0)))
-            ))).toList();
+                    URI.create(fromURI),
+                    this.uri(runContext, renderedTo + fromURI.substring(fromURI.lastIndexOf('/') + (renderedTo.endsWith("/") ? 1 : 0)))
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/fs/vfs/Uploads.java:62]: same thing, it misses a -1 check

**Review by brian-mulier-p — CHANGES_REQUESTED** [review]

File: src/main/java/io/kestra/plugin/fs/vfs/Upload.java (hunk 1 above)
Comment (loicmathieu) [reply src/main/java/io/kestra/plugin/fs/vfs/Upload.java:38]: In fact what's missing is to check that from is a Ketra URI, with that check we can blindly indexOd.

**Review by brian-mulier-p — APPROVED** [review]
LGTM

**Pull request merged**